aws/GitOps/clusters/optional-test-infra-prow/namespaces/prow/config.yaml

```diff
@@ -189,6 +189,21 @@ data:
           - image: public.ecr.aws/j1r0q0g6/kubeflow-testing:latest
             imagePullPolicy: Always
             command: ["/usr/local/bin/run_workflows.sh"]
+
+    postsubmits:
```

Comment:
Is the indentation correct here? I think everything needs to be moved back 4 spaces